components/src/core/channel-value/channel-value.component.scss

```diff
@@ -24,10 +48,26 @@
         font-weight: 600;
     }
 
+    .pxb-channel-value-value + .pxb-channel-value-units-suffix {
+        margin-left: 0.25em;
+    }
+
+    .pxb-channel-value-units-prefix + .pxb-channel-value-value {
+        margin-left: 0.25em;
+    }
+
+    .pxb-channel-value-value.pxb-channel-value-remove-space + .pxb-channel-value-units-suffix {
+        margin-left: 0;
+    }
+
+    .pxb-channel-value-units-prefix + .pxb-channel-value-value.pxb-channel-value-remove-space {
+        margin-left: 0;
+    }
+
     .pxb-channel-value-icon-wrapper {
         font-size: inherit;
         display: block;
-        margin-right: 0.25rem;
+        margin-right: 0.35em;
```

Comment:
Was this change intensional?  .035em is roughly 5.6 pixels whenever 1em=16px, doesn't fall into a multiple of 2.

Reply:
It was.

And 1em is different from 1rem. 1em would scale with the current font size. 

This is meant to prevent the `font-size` of the current channel value going up to, say, 72px, while the gap stays 4px wide.